database/user.go

```diff
@@ -455,6 +456,14 @@ func (db *DB) UserDelete(ctx context.Context, u *User) error {
 	if err != nil {
 		return errors.AddContext(err, "failed to delete user registry writes")
 	}
+	_, err = db.staticAPIKeys.DeleteMany(ctx, filter)
```

Comment:
why is this not a composed error? we want to try and delete as much as possible right? no reason to return on error but instead compose alle errors?

Reply:
That's one way to do it. We delete the user last here, so the client can retry the deletion until they succeed.

I would be fine with "delete the user first and then as much as possible".

Reply:
I think the f/u would be to mark the user for deletion and then retry indefinitely until all of their data is gone. Finally we remove the "mark".

We wouldn't want to have partial data stored.

Reply:
F/U: https://github.com/SkynetLabs/skynet-accounts/issues/136